fastlane/Fastfile

```diff
@@ -77,206 +77,20 @@ ENV['PUBLIC_CONFIG_FILE'] = VERSION_FILE_PATH
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane build_and_upload_internal [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane build_and_upload_internal
-  # bundle exec fastlane build_and_upload_internal skip_confirm:true
-  #####################################################################################
-  desc 'Builds and updates for distribution'
-  lane :build_and_upload_internal do |options|
-    unless options[:skip_prechecks]
-      ios_build_prechecks(
-        skip_confirm: options[:skip_confirm],
-        internal: false
-      )
-    end
-    ios_build_preflight unless options[:skip_prechecks]
-
-    internal_code_signing
-
-    gym(
-      scheme: 'Simplenote',
-      configuration: 'Distribution Internal',
-      workspace: WORKSPACE,
-      export_method: 'enterprise',
-      clean: true,
-      output_directory: OUTPUT_DIRECTORY_PATH,
-      export_team_id: TEAM_ID_ENTERPRISE,
-      export_options: {
-        method: 'enterprise',
-        provisioningProfiles: simplenote_provisioning_profiles(
-          root_bundle_id: "#{APP_STORE_BUNDLE_IDENTIFIER}.Internal",
-          match_type: 'InHouse'
-        )
-      }
-    )
-
-    File.rename(File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote.ipa'), File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote Internal.ipa'))
-
-    appcenter_upload(
-      api_token: EnvManager.get_required_env!('APPCENTER_API_TOKEN'),
-      owner_name: 'automattic',
-      owner_type: 'organization',
-      app_name: 'Simplenote',
-      file: File.join(OUTPUT_DIRECTORY_PATH, 'Simplenote Internal.ipa'),
-      notify_testers: false
-    )
-
-    sentry_upload_dsym(
-      auth_token: EnvManager.get_required_env!('SENTRY_AUTH_TOKEN'),
-      org_slug: 'a8c',
-      project_slug: 'simplenote-ios',
-      dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
-    )
-  end
-
   # Upload the localized metadata (from `fastlane/metadata/`) to App Store Connect
   #
   # @option [Boolean] with_screenshots (default: false) If true, will also upload the latest screenshot files to ASC
   #
   desc 'Upload the localized metadata to App Store Connect, optionally including screenshots.'
-  lane :update_metadata_on_app_store_connect do |options|
+  lane :update_metadata_on_app_store_connect do |with_screenshots: false|
     # Skip screenshots by default. The naming is "with" to make it clear that
     # callers need to opt-in to adding screenshots. The naming of the deliver
     # (upload_to_app_store) parameter, on the other hand, uses the skip verb.
-    with_screenshots = options.fetch(:with_screenshots, false)
     skip_screenshots = !with_screenshots
 
     upload_to_app_store(
       app_identifier: APP_STORE_BUNDLE_IDENTIFIER,
-      app_version: ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH),
+      app_version: release_version_current,
```

Comment:
This was required because of the `release-toolkit` 12.0 upgrade.

In hindsight, the upgrade was not required because `create_release_backmerge_pull_request` was already available in version 11, but it ought to have been done sooner or later, so might as well do it now.